docs/ReleaseNotes.md

```diff
@@ -1,20 +1,62 @@
 # Acuminator Release Notes
-This document provides information about fixes, enhancements, and key features that are available in Acuminator 1.3.
+This document provides information about fixes, enhancements, and key features that are available in Acuminator.
```

Comment:
@k-s-popova @dropsonic I think everything is ok but we shouldn't release the new version without ATR-217 for which there is a pull request already. Too many false alerts for DACs otherwise. And if we add this item then the description for it should be added to the release notes